### PR TITLE
skip import image plugin when no floating tags are present

### PR DESCRIPTION
**Pull request by rcerven**
Signed-off-by: Robert Cerven <rcerven@redhat.com>

### CODE REVIEW

File: atomic_reactor/util.py [hunk 1]
```diff
@@ -1369,9 +1369,11 @@ def get_primary_images(workflow):
...
     floating_images = workflow.tag_conf.floating_images
     if not floating_images:
-        floating_images = [
-            ImageName.parse(floating) for floating in
-            workflow.build_result.annotations['repositories']['floating']]
+        floating_results = workflow.build_result.annotations['repositories'].get('floating')
+        if floating_results:
+            floating_images = [
+                ImageName.parse(floating) for floating in
+                workflow.build_result.annotations['repositories']['floating']]
```
Comment (athos-ribeiro) [inline atomic_reactor/util.py]: this could be `floating_results` (and maybe it would all fit in a single line)

File: atomic_reactor/util.py [hunk 2]
```diff
@@ -1369,9 +1369,9 @@ def get_primary_images(workflow):
 def get_floating_images(workflow):
     floating_images = workflow.tag_conf.floating_images
     if not floating_images:
-        floating_images = [
-            ImageName.parse(floating) for floating in
-            workflow.build_result.annotations['repositories']['floating']]
+        floating_results = workflow.build_result.annotations['repositories'].get('floating')
```
Comment (twaugh) [inline atomic_reactor/util.py]: I think this should be `.get('floating', [])`?

(If so, can we make sure one of the test cases catches it?)

Comment (rcerven) [reply atomic_reactor/util.py]: added

**Comment by rcerven** [comment]
rn not updated: fix for unreleased feature

**Pull request merged**